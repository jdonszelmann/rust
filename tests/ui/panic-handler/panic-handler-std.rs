//@ normalize-stderr: "loaded from .*libcore-.*.rlib" -> "loaded from SYSROOT/libcore-*.rlib"

extern crate core;

use core::panic::PanicInfo;

#[panic_handler]
fn panic(info: PanicInfo) -> ! { //~ ERROR found duplicate lang item `panic_impl`
    loop {}
}

fn main() {}
