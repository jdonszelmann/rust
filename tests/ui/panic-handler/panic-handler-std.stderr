error[E0152]: found duplicate lang item `panic_impl`
  --> $DIR/panic-handler-std.rs:8:1
   |
LL | / fn panic(info: PanicInfo) -> ! {
LL | |     loop {}
LL | | }
   | |_^
   |
   = note: the lang item is first defined in crate `std` (which `panic_handler_std` depends on)
   = note: first definition in `std` loaded from $BUILD_DIR/x86_64-unknown-linux-gnu/stage1/lib/rustlib/x86_64-unknown-linux-gnu/lib/libstd-d7cc1867e59baf27.rlib
   = note: second definition in the local crate (`panic_handler_std`)

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0152`.
